src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -220,8 +241,49 @@ sample({
   target: $wallets,
 });
 
+sample({
+  clock: walletHidden,
+  fn: (wallet) => wallet,
+  target: hideWalletFx,
+});
+
+sample({
+  clock: hideWalletFx.doneData,
+  source: { wallets: $wallets, hiddenWallets: $hiddenWallets },
```

Comment:
Let's improve readability, same for the following `spreads`
```suggestion
  source: {
    wallets: $wallets,
    hiddenWallets: $hiddenWallets,
  },
```